felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/rapidsrivers/model/Event.kt

```diff
@@ -53,10 +54,11 @@ class Event(val event: EventName, private val jsonMessage: JsonMessage, val clie
     }
 
     fun createBehov(behov: BehovType, map: Map<DataFelt, Any>): Behov {
-        val forespoerselID = jsonMessage[Key.FORESPOERSEL_ID.str]
+        val forespoerselID = jsonMessage[Key.FORESPOERSEL_ID.str].takeUnless { it.isMissingOrNull() }
         return Behov(
             event,
             behov,
+            this.forespoerselId,
```

Comment:
er det noen forskjell her på this.foresoerselId og forespoerselID rett over?

Reply:
Men hvis det ikke har noe å si er det greit for meg 👍

Reply:
Veldig liten forskjell i hvert fall. Å ta det direkte fra jsonmessage er litt som et safeguard at noen har endret uten vilje forespørselId.  Jeg har ikke nok hjernekraft å tenke på det etter ferie :)